### PR TITLE
OPENEUROPA-355: Theming confirm password.

### PR DESCRIPTION
**Pull request by aritomelo**
It was needed to attach 'ecl-help-block' however the element is rendered by javascript so the approach took was via styling.

### CODE REVIEW

File: sass/components/_form.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    .is-good,
+    .is-strong {
+      background-color: $ecl-color-success;
+    }
+  }
+
+  /* @see @ec-europa/ecl-forms-help-blocks/_help-blocks.scss */
+  .password-strength .password-strength__title,
+  .password-confirm {
```
Comment (richardcanoe) [inline sass/components/_form.scss]: ecl-help-block should also be added to extend password-suggestions.

File: oe_theme.theme [hunk 2]
```diff
@@ -155,6 +155,11 @@ function _oe_theme_preprocess_input_label_wrapper(array &$variables) {
  * label is already rendered in the input template.
  */
 function oe_theme_preprocess_form_element(&$variables) {
+  $variables['is_password_confirm'] = FALSE;
+  if (in_array('password_confirm', explode('[', $variables['element']['#name']))) {
+    $variables['is_password_confirm'] = TRUE;
+  }
+
```
Comment (ademarco) [inline oe_theme.theme]: Is this necessary at all?

Comment (aritomelo) [reply oe_theme.theme]: No, it isn't. I will push a fix asap.

Comment (ademarco) [reply oe_theme.theme]: Ok thanks, I'll reopen the ticket then.

**Pull request merged**